data/json/items/gun/4570.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "id": "1895sbl",
+    "copy-from": "rifle_manual",
+    "type": "GUN",
+    "name": "Marlin 1895 SBL",
+    "description": "A handy but powerful lever-action rifle chambered for .45-70 Government. Designed for wilderness guides for defense against large predators such as grizzly bears, moose, and dinosaurs.",
+    "weight": 3650,
+    "volume": 9,
+    "price": 400000,
+    "to_hit": 1,
+    "bashing": 12,
+    "material": [ "steel", "wood" ],
+    "symbol": "(",
+    "color": "light_blue",
+    "ammo": "4570",
+    "dispersion": 200,
+    "durability": 8,
+    "clip_size": 7,
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ]
+    ],
+    "flags": [ "RELOAD_ONE" ]
+  },
+  {
+    "id": "bfr",
+    "copy-from": "pistol_revolver",
+    "type": "GUN",
+    "name": "Magnum Research BFR",
+    "name_plural": "Magnum Research BFR",
```

Comment:
```suggestion
```
The game will automatically add `s` to the `name`, and I think `Magnum Research BFRs` is a more proper plural name than without `s`.